Automattic/jetpack
# CRM: Remove assumption that conditional avatar input exists

We assume that two inputs always exists but it requires "avatarmode" to be set to `2` before they appear, which means the SMS prompt/trigger that comes later in `zeroBSCRMJS_editContactInit` doesn't work because due to an error being triggered.

Fixes https://github.com/Automattic/zero-bs-crm/issues/2881

## Proposed changes:

* Add conditional check around the `zbsc_profile-picture-file` and `zbsc_remove-profile-picture-button` input logic.

### Other information:


## Jetpack product discussion



https://github.com/Automattic/zero-bs-crm/issues/2881

## Does this pull request change what data or activity we track or use?



No.

## Testing instructions:






* Spin up fresh site
* Install CRM
* Go to /wp-admin/admin.php?page=zbs-add-edit&action=edit&zbstype=contact
* Check developer console

## Screenshot(s)

![218151706-5d1cec8d-2a8f-4c9f-b6dd-b2c79e1a0bda](https://user-images.githubusercontent.com/3846700/220486874-a4433c60-7068-4feb-8bd0-6519e9ff1924.jpeg)
